testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/client/policies/ClientPoliciesExecutorTest.java

```diff
@@ -560,12 +562,18 @@ public void testSecureRequestObjectExecutor() throws Exception {
 
         // check whether request object not yet being processed
         requestObject = createValidRequestObjectForSecureRequestObjectExecutor(clientId);
-        requestObject.nbf(requestObject.getNbf() + 600);
+        requestObject.nbf(requestObject.getNbf() + allowedClockSkew + 2);
         registerRequestObject(requestObject, clientId, Algorithm.ES256, false);
         oauth.openLoginForm();
         assertEquals(OAuthErrorException.INVALID_REQUEST_OBJECT, oauth.getCurrentQuery().get(OAuth2Constants.ERROR));
         assertEquals("Request not yet being processed", oauth.getCurrentQuery().get(OAuth2Constants.ERROR_DESCRIPTION));
 
+        // nbf ahead within allowed clock skew
+        requestObject = createValidRequestObjectForSecureRequestObjectExecutor(clientId);
+        requestObject.nbf(requestObject.getNbf() + allowedClockSkew - 2);
```

Comment:
Minor: Is it possible to use bigger interval? For example something like ` ... + allowedClockSkew - 10` ? I afraid that if it is only 2 seconds, the test can fail in some slow environments.

Reply:
Yes, I will use the bigger interval as you suggested.